force-app/main/default/classes/HOT_UpdateExistingRecords.cls

```diff
@@ -1,10 +1,22 @@
 public class HOT_UpdateExistingRecords implements Database.Batchable<sObject> {
     public Database.QueryLocator start(Database.BatchableContext bc) {
         Datetime now = Datetime.now();
-        return Database.getQueryLocator('SELECT Id FROM HOT_Request__c');
+        return Database.getQueryLocator('SELECT HOT_Interpreters__c FROM WorkOrder WHERE Status != \'Completed\' AND Status != \'Partially Complete\' AND Status != \'Dispatched\'');
     }
 
-    public void execute(Database.BatchableContext bc, List<HOT_Request__c> records) {
+    public void execute(Database.BatchableContext bc, List<WorkOrder> records) {
+        List<WorkOrder> workOrdersWithInterpreterFieldToBeBlank = new List<WorkOrder>();
+        for (WorkOrder workOrder : records) {
+            workOrder.HOT_Interpreters__c = '';
+            workOrdersWithInterpreterFieldToBeBlank.add(workOrder);
```

Comment:
Denne bare blanker ut feltet.. Det vil vel ikke trigge at triggere fyller det ut?

Reply:
Åja, trodde vi bare skulle blanke ut for gamle bestillinger og la koden kjøre for nye.

Reply:
Nei, feltet skal alltid ha en verdi om WorkOrder.Status = Dekket/Delvis Dekket/Tildelt. Vi skjuler det kun for bruker når vi henter det ut.